### DISCUSSION TITLE
Avalonia 11 controls intellisense.

### DISCUSSION THREAD

**Discussion by Quetzalcoatl2435** [AvaloniaUI/Avalonia#13675] [Q&A]
Avalonia 11 has an internal XAML name reference generator, but intellisense doesn't detect it?
It throws a CS0103: The name does not exists in the current context.

```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:AvaloniaApplication1.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:views="clr-namespace:AvaloniaApplication1.Views"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="AvaloniaApplication1.Views.MainWindow"
        Icon="/Assets/avalonia-logo.ico"
        Title="AvaloniaApplication1">
	<StackPanel VerticalAlignment="Center" HorizontalAlignment="Center">
		<Label Name="lbMsg" />
		<Button Name="btnTest" />
	</StackPanel>
</Window>
```

![Screenshot 2023-11-21 161233](https://github.com/AvaloniaUI/Avalonia/assets/81870114/cd970e69-4f4b-40bf-8fac-830001171a92)

**Comment by timunie** [comment]
sometimes the source gen cache is blocking it. You may need to remove the files by hand and rebuild the solution

**Reply by Quetzalcoatl2435** [reply]
Which files would that be? I've tried deleting cache files in obj/, but still no change.

**Reply by timunie** [reply]
source generated files. They are somewhere else depending on the ide. Check your ide docs for where to find them.

**Reply by Quetzalcoatl2435** [reply]
Ah, that's makes more sense.

Under-the-hood VS documentation is sparse, though. I couldn't find anything related to the specifics of caches, and deleting every VS cache I could find didn't change anything either, so I'm giving up for now.

I'll update this whenever I find it/can get it to work. Or if anyone else has a solution, please chime in. :)

Thanks.
